code-challenges.test.js

```diff
@@ -70,4 +190,23 @@ const accountTransactions3 = []
 
 // b) Create the function that makes the test pass.
 
+const sumOf = (addition) => {
+  let placeHolder = 0
+ for(let i=0; i<addition.length;i++) {
+     placeHolder = addition[i] + placeHolder
+     addition[i] = placeHolder
+ } return addition
+}
+
+// Test Suites: 1 passed, 1 total
+// Tests:       1 passed, 1 total
+
 // Pseudo code:
+// 1. Create a function called sumOf, has 1 parameter called addition.
+      // 1a. Declare a new variable named placeHolder equal to 0.
+// 2. Create a for loop to iterate over each index.
+      // 2a. PlaceHolder is within our local scope.
+      // 2b. Have placeHolder equal to our parameter with backet notation to signal index [i], arithmetic operator + to add placeHolder.
+      // 2c. Have addition[i] = placeHolder, so it can iterate each index within the loop to add the last index value to the current one.
+// 3. Return addition outside of the for loop.
```

Comment:
Good logic here. I would mention how placeHolder is the accumulated sum that is being updated each iteration.